samples/ballerina-by-example/examples/constrained-json/constrained-json.bal

```diff
@@ -18,90 +8,19 @@ struct Person {
-        transform {
-            //Split name from " " to separate first and last names.
-            string[] _temp = strings:split(people[index].name, " ");
-            employeeJson.first_name = _temp[0];
-            employeeJson.last_name = _temp[1];
-            employeeJson.age = people[index].age;
-            employeeJson.address = people[index].city;
-            // Accessing an invalid field that is not a field in Employee struct will give an error.
-            // E.g. : employeeJson.city will give an error as "unknown field 'city' in json with struct constraint 'Employee'".
-        }
-
-        // Set the new employeeJson as the payload.
-        response:setJsonPayload(res, employeeJson);
-        response:send(res);
-    }
-
-    @http:resourceConfig {
-        methods:["POST"],
-        path:"/update"
-    }
-    @doc:Description {value:"Resource to update person by index which can be invoked with /people/update?index={index}"}
-    resource updatePerson (http:Request req, http:Response res) {
-        //Get the json payload from the request message m.
-        json j = request:getJsonPayload(req);
-        var jsonPerson, _ = (json<Person>)j;
-        map params = request:getQueryParams(req);
-        var index, _ = (int)params.index;
-        // Define an error that can be used during conversion from json to person struct.
-
-        transform {
-            people[index].name, _ = (string)jsonPerson.name;
-            people[index].age, _ = (int)jsonPerson.age;
-            people[index].city, _ = (string)jsonPerson.city;
-            // Accessing an invalid field that is not a field in Employee struct will give an error.
-            // E.g. : employeeJson.address will give an error as "unknown field 'address' in json with struct constraint 'Person'"
-        }
-
-        //Set the status code to 202 Accepted.
-        response:setStatusCode(res, 202);
-        response:send(res);
-    }
-
-    @http:resourceConfig {
-        methods:["GET"],
-        path:"/"
-    }
-    @doc:Description {value:"Get all people which can be invoked as /people/"}
-    resource GetPeople (http:Request req, http:Response res) {
-        // Define empty json array.
-        json jsonResponse = [];
-        int index = 0;
-        while (index < lengthof people) {
-            // Convert each person from people array to a JSON and set to the array.
-            jsonResponse[index], _ = <json>people[index];
-            index = index + 1;
-        }
-
-        // set the people array as the json response in the message
-        response:setJsonPayload(res, jsonResponse);
-        response:send(res);
-    }
+function main (string[] args) {
+    // Create a new json which is constrained by Person struct. Only the fields defined in the Person struct can be
+    // accessed. If we try to access a non existing field, it will produce a compilation error.
```

Comment:
Please make sure code lines are note greater than 85. Otherwise it will be out of the frame in the generated html file